src/utils/logger.js

```diff
@@ -37,4 +21,31 @@ const logger = winston.createLogger({
   rejectionHandlers: transports,
 });
 
+if (process.env.SMTP_PASSWORD) {
```

Comment:
SMTP servers can be set up to send without password authentication, for example if they are on the same private network. It thus seems a potential legit setup for `SMTP_PASSWORD` to be `undefined`. In order to keep this relevant check and warning, I suggest that we check explicitly if it is `undefined` rather than falsy, so that users having no password authentication can set `SMTP_PASSWORD` to the empty string 🙂